cmd/libs/go2idl/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -78,16 +79,31 @@ func (g *genFakeForType) GenerateType(c *generator.Context, t *types.Type, w io.
 	pkg := filepath.Base(t.Name.Package)
 	const pkgTestingCore = "k8s.io/kubernetes/pkg/client/testing/core"
 	namespaced := !(types.ExtractCommentTags("+", t.SecondClosestCommentLines)["nonNamespaced"] == "true")
+	canonizeGroup := func(group string) string {
```

Comment:
I think you mean canonicalize.

https://en.wikipedia.org/wiki/Canonization_(disambiguation)

> Sometimes, the term is used incorrectly to refer to canonicalization, i.e., finding a canonical form